src/api/pipelineRuns.js

```diff
@@ -78,6 +78,7 @@ export function deletePipelineRun({ name, namespace }) {
 export function createPipelineRun({
   namespace,
   pipelineName,
+  pipelineRunName,
```

Comment:
we should add the default value here so it's available to all consumers of the API
```diff
-  pipelineRunName,
+  name = `${pipelineName}-run-${Date.now()}`,
```

Reply:
@AlanGreene Thank you :D I reflected your suggestion in https://github.com/tektoncd/dashboard/pull/2345/commits/5448a2166e592bdc15d097e4d29d3423463c2c0b